lib/ansiblelint/__init__.py

```diff
@@ -68,7 +68,7 @@ def matchlines(self, file, text):
             message = None
             if isinstance(result, six.string_types):
                 message = result
-            matches.append(Match(prev_line_no+1, line,
+            matches.append(Match(prev_line_no + 1, line,
```

Comment:
This is not a docstring

Reply:
I guess that is obvious **but** flake8-docstrings complained about it, so I fixed it.

Reply:
really? it's hard to believe. are you sure it wasn't just flake8 itself? (w/o plugins)